src/elements/Accordion/Accordion.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Accordion
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import Grid from '../Grid/Grid';
+import { isMobile } from '../../util/helpers';
+
+class Accordion extends Component {
+	static propTypes = {
+		children: PropTypes.node.isRequired,
+		title: PropTypes.node.isRequired,
+		className: PropTypes.string,
+		expanded: PropTypes.bool,
+		notification: PropTypes.bool // this is used to style accordion as a notification
+	};
+
+	constructor(props) {
+		super(props);
+
+		if (this.props.expanded) {
```

Comment:
Let's turn this into a ternary on the property in the state object itself instead of the if/else statement.

Also, let's change the word `expanded` to be `defaultExpanded` so that it's verbose and clear that it's not meant to be changed from the parent.